sample/allofhands_hawaii2016/sessions

```diff
@@ -0,0 +1,3723 @@
+[
+
+	{
+		"short_abstract": "",
+		"speakers": [],
+		"subtitle": null,
+		"language": "english",
+		"title": "Hawaii All Hands - Plenary Session",
+		"track": {
+			"id": 1,
+			"title": "All Hands"
+		},
+
+		"start_time": "2016-12-06T19:00:00",
+		"session_type": {},
+		"comments": "",
+		"long abstract": null,
+		"slides": null,
+		"state": "accepted",
+		"video": null,
+		"end_time": "2016-12-06T21:00:00",
+		"timezone": "UTC",
+		"microlocation": {
+			"id": 19,
+			"name": "Monarchy Ballroom"
+		},
+		"audio": null,
+		"signup_url": null,
+		"id": 7
+	},
+
+
+	{
+		"short_abstract": "",
+		"speakers": [{
+			"id": 105,
+			"name": "Sean White",
+			"organisation": null
+		}],
+		"subtitle": null,
+		"language": "english",
+		"title": "IT: Application Services - Service Strategy Pivot",
+		"track": {
+			"id": 3,
+			"title": "IT"
+		},
```

Comment:
@shreyasnbhat these are the titles there in the Mozilla database. I'm not sure if we want to change that as the aim is to re-create the Mozilla event. Let's wait for others to review I think.